src/lbaf/IO/lbsVTDataWriter.py

```diff
@@ -134,6 +135,20 @@ def __create_tasks(self, rank_id, objects, migratable):
             user_defined = o.get_user_defined()
             if user_defined:
                 task_data["user_defined"] = dict(sorted(user_defined.items()))
+            else:
+                task_data["user_defined"] = dict()
+
+            task_data["user_defined"]["object_memory"] = o.get_size()
```

Comment:
Can you do the same for the object-level QOIS? @cwschilly

Reply:
Done